app/app.js

```diff
@@ -1232,11 +1232,42 @@ var AppComponent = React.createClass({
       return null;
     }
 
+    var mostRecentUpload = _.sortBy(_.where(data, {type: 'upload'}), function(d) {
+      return Date.parse(d.time);
+    }).reverse()[0];
+    var timePrefsForTideline;
+    if (!_.isEmpty(mostRecentUpload) && !_.isEmpty(mostRecentUpload.timezone)) {
+      try {
+        sundial.checkTimezoneName(mostRecentUpload.timezone);
+        timePrefsForTideline = {
+          timezoneAware: true,
+          timezoneName: mostRecentUpload.timezone
+        };
+      }
+      catch(err) {
+        app.log(err);
```

Comment:
so if I don't have a timezone and there isn't one from the query params what is the expected behaviour?